src/error-handler.ts

```diff
@@ -1,17 +1,16 @@
 import { APIGatewayProxyResult } from 'aws-lambda';
 import { consoleAwsLogger } from './console-aws-logger';
-import { AwsLogger } from './models/aws-logger.model';
-import { ResponseEntity, toApiGatewayProxyResult } from './models/response-entity.model';
+import { AwsLogger, ResponseEntity, toApiGatewayProxyResult } from './models';
 
 export type Predicate = (input: any) => boolean;
-export type ResponseProvider = (input: any) => ResponseEntity;
+export type ResponseProvider = (input: any) => ResponseEntity<any>;
```

Comment:
If we are being explicit then I would suggest replacing any with unknown. That kind of counts for all the any's present in the error-handler.

I did a quick search and replace in this file on all any's with unknown, TS still compiles without errors.